shared/components/Community/index.js

```diff
@@ -9,6 +9,16 @@ import MailingList from '../MailingList';
 import JoinSlack from '../JoinSlack';
 import FutureEvents from '../FutureEvents';
 
+const usefulLinks = [
```

Comment:
Nice :) These work perfectly when given data. Whats the plan with these when they write the copies, new pages from UX and Design?